examples/move/coin/sources/regcoin.move

```diff
@@ -3,25 +3,43 @@
 
 //docs::#regulate
 module examples::regcoin {
-    use sui::coin::{Self, DenyCap};
+    use sui::coin::{Self, DenyCapV2};
     use sui::deny_list::{DenyList};
 
     public struct REGCOIN has drop {}
 
     fun init(witness: REGCOIN, ctx: &mut TxContext) {
-        let (treasury, deny_cap, metadata) = coin::create_regulated_currency(witness, 6, b"REGCOIN", b"", b"", option::none(), ctx);
+        let (treasury, deny_cap, metadata) = coin::create_regulated_currency_v2(
```

Comment:
I think this is fine? Although it will be possibly weird for people following this example on mainnet/testnet before this change hits those networks.

Reply:
Ah, yeah let me maybe just revert this and allow deprecation? 
I didn't understand immediately that it was being used for docs